policies/deprecation-release-policy.md

```diff
@@ -0,0 +1,77 @@
+# Design System Library Deprecation and Release Policy
+
+## Overview
+
+This policy outlines the procedures and timelines for deprecating components, utilities, and other features in the design system library. It also details the release process to ensure smooth transitions and maintain the stability and reliability of the library.
+
+## Deprecation Policy
+
+### Definition of Deprecation
+
+Deprecation is the process by which a component, utility, or feature is marked as obsolete and is scheduled for removal in a future release. Deprecation serves as a warning to users that they should transition to the recommended alternatives.
+
+### Deprecation Guidelines
+
+1. **Announcement**: Deprecation of a component, utility, or feature will be announced in the release notes of the version in which the deprecation first occurs. It will include:
+
+    - The rationale for deprecation.
+    - Suggested alternatives or migration paths.
+    - The timeline for removal.
+
+2. **Documentation**:
+
+    - All deprecated items will be clearly marked in the documentation with a "Deprecated" label.
+    - Documentation will include information on the deprecation and guidance on transitioning to alternatives.
+    - Our documentation about the component is displayed on the [Service Manual](https://service-manual.ons.gov.uk/).
+
+3. **Codebase**:
+    - Deprecated items will include a deprecation warning in the console, if applicable, to alert developers during development.
+    - Deprecated code will be maintained and supported (excluding new features) until its removal.
+
+### Deprecation Timeline
+
+1. **Initial Announcement**: Deprecated items will be announced and documented in a minor release.
+2. **Grace Period**: There will be at least two subsequent minor releases or a minimum of six months (whichever is longer) before the deprecated item is removed.
+3. **Removal**: Deprecated items will be removed in the next major release following the grace period.
+
+## Release Policy
```

Comment:
I would put the release policy first, then deprecation info - so it follows a more 'general to specific' trajectory

Reply:
Done :)